gammapy/time/lombscargle_gammapy.py

```diff
@@ -0,0 +1,171 @@
+# from lombscargle
+from astropy.stats import LombScargle
+# from PyAstronomy.pyTiming.pyPeriod import Gls
+# from PyAstronomy.pyTiming import pyPeriod
+import numpy as np
+# import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib import rc
+from matplotlib import rcParams
+plt.style.use('ggplot')
+# import pylab
+from scipy.stats import beta
+from scipy import optimize
+
+__all__ = [
+    'lombscargle',
+    'plotting',
```

Comment:
How important is this `plotting` function?
Maybe it can be removed, or if you want to keep it, please rename to `lombscargle_plot`.
Otherwise people will not know what it is when it shows up in the `gammapy.time` namespace.